internal/pkg/service/stream/config/config_test.go

```diff
@@ -331,80 +330,37 @@ func TestTableSinkConfigPatch_ToKVs(t *testing.T) {
     "validation": "min=0,max=256"
   },
   {
-    "key": "storage.level.local.encoding.inputBuffer",
+    "key": "storage.level.local.encoding.encoder.type",
     "type": "string",
-    "description": "Max size of the buffer before compression, if compression is enabled. 0 = disabled",
-    "value": "1MB",
-    "defaultValue": "1MB",
+    "description": "Encoder type.",
+    "value": "csv",
+    "defaultValue": "csv",
     "overwritten": false,
     "protected": true,
-    "validation": "maxBytes=16MB"
+    "validation": "required,oneof=csv"
   },
   {
-    "key": "storage.level.local.encoding.outputBuffer",
+    "key": "storage.level.local.encoding.inputBuffer",
     "type": "string",
-    "description": "Max size of the buffer before the output. 0 = disabled",
+    "description": "Max size of the buffer before compression, if compression is enabled. 0 = disabled",
     "value": "1MB",
     "defaultValue": "1MB",
     "overwritten": false,
     "protected": true,
     "validation": "maxBytes=16MB"
   },
   {
-    "key": "storage.level.local.volume.allocation.enabled",
-    "type": "bool",
-    "description": "Allocate disk space for each slice.",
-    "value": true,
-    "defaultValue": true,
-    "overwritten": false,
-    "protected": true
-  },
-  {
-    "key": "storage.level.local.volume.allocation.relative",
-    "type": "int",
-    "description": "Allocate disk space as % from the previous slice size.",
-    "value": 110,
-    "defaultValue": 110,
-    "overwritten": false,
-    "protected": true,
-    "validation": "min=100,max=500"
-  },
```

Comment:
I removed the disk allocation from the customizable settings. If it is needed in the future, it can be added.

This made refactoring easier.

Reply:
I think that there are none circumstances, that's why I threw it away.